searx/engines/startpage.py

```diff
@@ -127,6 +127,9 @@
 # engine dependent config
 categories = ['general', 'web']
 paging = True
+max_page = 9
```

Comment:
I can request up to 18 pages on startpage: https://www.startpage.com/sp/search?query=the&cat=web
related: https://github.com/searxng/searxng/issues/2982#issuecomment-1810679732

could you set to max 20 pages just to be safe?